create-demo-setup.yml

```diff
@@ -20,8 +20,8 @@
         name: 'azure'
       vars:
         ssh_key_path: "{{ ssh_key | default(lookup('env', 'HOME') + '/.ssh/id_rsa.pub')}}"
-        #ssh_key_path: "{{ lookup('file', '/home/hcheruku/.ssh/1643057028_7780352.pub') }}"
         dns_name: "azuredemos.ansiblemiddleware.com" # This is optional
+        managed_mariadb: true
```

Comment:
My assumption is that we would want to spin up our own VM servers (as in the diagram) by default and have the option to go managed.

so `managed_mariadb: false`

Reply:
sure, I can make it false by default.